osi_hostvehicledata.proto

```diff
@@ -2,31 +2,356 @@ syntax = "proto2";
 
 option optimize_for = SPEED;
 
+import "osi_version.proto";
 import "osi_common.proto";
+import "osi_object.proto";
 
```

Comment:
Got following warning at the last CI unit test regarding osi_object.proto
`osi_hostvehicledata.proto: warning: Import osi_object.proto but not used.`

Reply:
Good point, I deleted the import!